### PR TITLE
Some easy issues

### PR DESCRIPTION
**Pull request by keatonwalker**
Handled most of the issues marked easy and fixed some errors.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -67,7 +67,12 @@ module.exports = function (grunt) {
             }
         },
         connect: {
-            uses_defaults: {}
+            main: {
```
Comment (keatonwalker) [inline Gruntfile.js]: Yes this was for development but the new setup we have makes it unnecessary.

File: src/app/location/Station.js [hunk 2]
```diff
@@ -141,13 +142,15 @@ define([
                 this.fGroup = new L.FeatureGroup().addTo(this.vMap.map);
                 this.pointDef.setMap(this.vMap.map, this.fGroup);
 
-                var that = this;
-                this.vMap.map.on('moveend', function () {
```
Comment (keatonwalker) [inline src/app/location/Station.js]: It was the source of a bug. The map extent of the main map is set on successful station submit but I will set it when the dialog is closed as well.

**Comment by keatonwalker** [comment]
I have fixed everything I have noticed so far. Let me know if you are ok with merging this.

**Comment by stdavis** [comment]
I had to make a fair amount of path modifications to get the script tests to work. I'm sure that we could make this better in the future but I think that it's good enough for now.

**Pull request merged**